src/test/compile-fail/wf-static-method.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2015 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// check that static methods don't get to assume `Self` is well-formed
+
+// TODO: I (arielb1) isn't sure whether this is a bug or feature.
```

Comment:
Note tidy error: /home/travis/build/rust-lang/rust/src/test/compile-fail/wf-static-method.rs:13: TODO is deprecated; use FIXME